### ISSUE TITLE
Maven Central Publishing

### ISSUE THREAD

**Issue by longshorej** [ilya-epifanov/akka-dns#10]
Would you be willing to publish this library to Maven Central? It reduces the friction for adopters as they don't need to add custom resolvers to their project.

I'd be happy to assist if you're open to the idea, please let me know.

**Comment by guizmaii** [comment]
The repo seems to be (very) unmaintained. Maybe Lightbend should propose a fork, @longshorej ?

**Comment by ilya-epifanov** [comment]
@guizmaii yep, I can only merge PRs for this one, BTW @huntc from Typesafe has full access to this repo

**Comment by huntc** [comment]
I’m no longer at Typesafe/Lightbend, but I’m happy to help out.

**Comment by guizmaii** [comment]
Hi @huntc,

Is it possible for you to release a new version with the updated libs ? 

thanks,
Jules

**Comment by huntc** [comment]
Hey @ilya-epifanov - would you be able to grant @longshorej access to this repo as he has taken over my responsibilities at Lightbend. I don’t appear able to grant him access. Cheers.

**Comment by ilya-epifanov** [comment]
@longshorej, welcome to the party :)
Please check out https://github.com/ilya-epifanov/akka-dns/invitations

**Comment by longshorej** [comment]
Thanks! I'll try to get a release out this week. Will update this thread when it's out. Maven Central TBD as I have to coordinate and confirm plans with a few colleagues.

**Comment by ktoso** [comment]
Hi everyone interested in DNS and async stuff ;-)
I'm working to update Akka's own DNS resolver to be properly async and "support all the things".

I'll take some inspiration from the excellent work done here, thanks a lot @ilya-epifanov everyone who contributed to the project! It's time for Akka itself to "proper" about its DNS handling :-)

I'll make sure to do proper kudos / shout out during the release of course, thanks a lot @ilya-epifanov for your work so far, once I do the PR would be nice if you could review then :-)
